jax/_src/numpy/lax_numpy.py

```diff
@@ -1415,6 +1415,24 @@ def unravel_index(indices, shape):
   idx = clipped_indices % cumulative_sizes[:-1] // cumulative_sizes[1:]
   return tuple(idx)
 
+@_wraps(np.resize)
+def resize(a, new_shape):
+  if isinstance(new_shape, (int, integer)):
+    new_shape = (new_shape,)
```

Comment:
We have the `_ensure_index_tuple` utility for this check, which is a bit more robust.